packages/nimble-components/src/number-field/specs/IxD.md

```diff
@@ -0,0 +1,160 @@
+
+## Requirements
+
+1. Support locale-specific formatting for decimal values (e.g., 10212,456 in
+   German).
+2. Allow specifying decimal precision (e.g., 2 places → 123.45).
+3. Provide validation-only mode for max/min, supporting both coercion and
+   non-coercion on blur.
+
+### Anatomy
+
+![ ](spec-images/number-field-anatomy.png)
+
+| Element             | Description                                                                                      |
+| ------------------- | ------------------------------------------------------------------------------------------------ |
+| Label               | Name for the field.                                                                              |
+| Required            | Indicates whether the field is required or optional.                                             |
+| Placeholder         | Shows an example of the data required in the field or a description of the purpose of the field. |
+| Numeric value       | Current control value.                                                                           |
+| Increment/Decrement | Changes numeric value by the step size.                                                          |
+| Error status icon   | Appears when a user has entered invalid data in a field.                                         |
+| Error text          | Appears when a user has entered invalid data in a field.                                         |
+
+### Related Components
+
+- [Number Field](https://nimble.ni.dev/storybook/index.html?path=/docs/components-number-field--docs)
+
+## Design
+
+### Configuration
+
+Proposed attributes:
+
+| Attribute               | Description                                                                                                               |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| maximum-fraction-digits | Maximum number of digits after the decimal separator.                                                                     |
+| minimum-fraction-digits | Minimum number of fraction digits. Values with fewer digits will be right-padded with zeros to meet the specified length. |
+| min                     | The lowest permitted value.                                                                                               |
+| max                     | The greatest permitted value.                                                                                             |
+
+### Behavior
+
+#### Mouse Interactions
+
+Hovering over the field input indicates interactivity.
+
+Clicking on the field input focuses the input and places a caret. If no value
+exists, the caret is placed at the far left. Otherwise, it is placed where
+clicked.
+
+Double-clicking within the field input selects the closest contiguous
+characters.
+
+Triple-clicking within the field input selects all characters.
```

Comment:
It took me a while to figure out how you can have characters that aren't contiguous. Might be worth explicitly pointing out that `+`s, `-`s, and decimal separators (`.`/`,`) break contiguity. (`e`s are contiguous with digits).